src/classes/CW_MessagingApprovalController.cls

```diff
@@ -0,0 +1,469 @@
+/**
+* File Name   :    CW_MessagingApprovalController.cls
+* Description :    Page for approving messaging records
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 04/08/2014 Suneetha Jasthi	 B-122329 Messaging dates approval process
+*/
+public with sharing class CW_MessagingApprovalController 

```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.